kyuubi-common/src/main/scala/org/apache/kyuubi/Utils.scala

```diff
@@ -24,8 +24,11 @@ import java.util.{Properties, UUID}
 
 import scala.collection.JavaConverters._
 
+import org.apache.commons.lang.SystemUtils
 import org.apache.hadoop.security.UserGroupInformation
 
+
+
```

Comment:
these are unnecessary, can we remove?

Reply:
OK. I have no problem testing it on my env, I will test it on Linux before committing.

Reply:
Could you help delete the user information of 003006? I just found that I used the wrong user when committing.

Reply:
you can set the right username locally,   then  re-push this by `push -f`